src/chatClient.cpp

```diff
@@ -3645,6 +3645,14 @@ void ContactList::syncWithApi(mega::MegaUser& user)
         {
             contact->mEmail = user.getEmail();
             client.db.query("update contacts set email = ? where userid = ?", contact->email(), handle);
+            client.userAttrCache().onUserAttrChange(user);
```

Comment:
It would be good to avoid re-fetch the email by sending a new API command. Instead, you could update the value here immediately, from the information received in the `MegaUser`. Only if it's relatively simple, otherwise email changes doesn't happen very often, so it would be fine.

Reply:
It would require to tweak the update of the `userAttrCache`, not clean. Let's fetch a fresh email in these cases.